pydata-warsaw/category.json

```diff
@@ -0,0 +1,3 @@
+{
+  "title": "PyData Warsaw"
```

Comment:
@Daniel-at-github , could this (and the folder name) also include the year?

Reply:
This are the normal group reunion videos, not the PyData Warsaw 2017 conference.
I supposed the the two can coexist, but no, they can't. See the error:
~~~
CRITICAL: ValueError: time data '' does not match format '%Y-%m-%d %H:%M:%S %z'
~~~
What can we do in this cases? ... change the name?

Reply:
> @redapple could this (and the folder name) also include the year?
No. PyData Warsaw 2017 conference is out, and is a different thing.

Reply:
I would suggest using something like "PyData Warsaw Meetups" and link it to
https://www.meetup.com/PyData-Warsaw/events/